control-plane/tests/test_cli_inspection.py

```
@@ -1315,6 +1356,477 @@ def test_cli_renders_case_detail_with_evidence_provenance_and_cited_advisory_out
+            detail_payload["case_record"]["reviewed_context"]["triage"]["disposition"],
+            "business_hours_handoff",
+        )
+
+    def test_long_running_runtime_surface_records_bounded_operator_casework_actions(self) -> None:
+        store, _ = make_store()
+        service = AegisOpsControlPlaneService(
+            RuntimeConfig(
+                host="127.0.0.1",
+                port=0,
+                postgres_dsn="postgresql://control-plane.local/aegisops",
+                wazuh_ingest_shared_secret="reviewed-shared-secret",
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to argument: "wazuh_ingest_shared_secret"

(S106)